cmake/init_target.cmake

```diff
@@ -10,7 +10,9 @@ function(init_target target_name) # init_target(my_target folder_name)
     elseif (APPLE)
```

Comment:
```diff
-    if (WIN32)
-        target_compile_features(${target_name} PUBLIC cxx_std_17)
-    elseif (APPLE)
+    if (APPLE)
```

Reply:
Why do I need to change Windows definitions?

Reply:
Because they're both became cxx_std_17

Reply:
Yes, but they may be changed in the future.

Reply:
And these lines could be returned in the future :thinking:

Reply:
Done.